cves/CVE-2019-9721.yml

```diff
@@ -144,10 +153,10 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: There were no unit_tests around this area of code
+  fix: false
+  fix_answer: No fate Suite
```

Comment:
http://coverage.ffmpeg.org/ will be a good resource for you to double check this.